.circleci/config.yml

```diff
@@ -42,12 +42,34 @@ jobs:
             - ~/workbench/api/build/exploded-api/WEB-INF/lib/
           key: api-cache-{{ checksum "~/workbench/api/build.gradle" }}
       - deploy:
-          name: Deploy to gcloud test project
+          name: Maybe deploy to test/staging
           working_directory: ~/workbench/api
           command: |
-            ./project.rb circle-deploy \
-              --project all-of-us-workbench-test --creds-file circle-sa-key.json
-  
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              ./project.rb deploy \
+                --project all-of-us-workbench-test \
+                --account=circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
+                --version circle-ci-test \
+                --promote
+            elif [ "${CIRCLE_TAG}" == "v*" ]; then
+             ./project.rb deploy \
+               --project all-of-us-rw-staging \
+               --account=circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
+               --version "${CIRCLE_TAG}" \
+               --promote
+            else
+              echo "Not master branch or tagged release, skipping deploy"
+            fi
+      - run:
+          name: Scan dependencies for vulnerabilities
+          working_directory: ~/workbench/api
+          command: |
+            if [ -n "${CIRCLE_TAG}" ]; then
```

Comment:
(Not sure if we want to do this dependency check on all tags or just the v* ones?)